doc/api/tls.md

```diff
@@ -884,6 +907,17 @@ changes:
     verified against the list of supplied CAs. An `'error'` event is emitted if
     verification fails; `err.code` contains the OpenSSL error code. **Default:**
     `true`.
+  * `pskCallback(hint)` {Function} When negotiating TLS-PSK, this optional
+    function is called with the identity hint provided by the server. If the
+    client should continue to negotiate PSK ciphers, the return value of this
+    function must be an object in the form `{psk: <string>, identity:
+    <string>}`.
+    Note that PSK ciphers are disabled by default, and using TLS-PSK thus
+    requires explicitly specifying a cipher suite with the `ciphers` option.
+    Additionally, it may be necessary to disable `rejectUnauthorized` if a
+    client attempts to specify a certificate for the session.
```

Comment:
I don't think it should be necessary to use `rejectUnauthorized` if the user wants to use only PSK. Though currently, this fails with "unable to get issuer certificate" if I try to remove it in tests but IIUC it should work without `rejectUnauthorized`.
I'll investigate this further as this is an openssl error, so perhaps I missed something in the configuration of openssl.

Reply:
Refer to https://github.com/nodejs/node/pull/23188#issuecomment-447900198 and

https://github.com/nodejs/node/pull/23188/files#diff-f6e3a86962eaf0897ab59e88b418e64fR1092 :

    It will be necessary to provide a custom [`tls.checkServerIdentity()`][]
    for the connection as the default one will try to check hostname/IP
    of the server against the certificate but that's not applicable for PSK
    because there won't be a certificate present.